### PR TITLE
limit 5 crypto

### PR DESCRIPTION
**Pull request by Jocelyne28**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function App() {
 
 
   const getData = async ()  => {
-    const res = await axios.get("https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1");
+    const res = await axios.get("https://api.coingecko.com/api/v3/coins/markets?vs_currency=USD&order=market_cap_desc&per_page=5&page=1");
```
Comment (PkBadger) [inline src/App.js]: Just make sure that the list don't change

File: src/App.js [hunk 2]
```diff
@@ -10,9 +10,17 @@ function App() {
...
+    const res = await axios.get("https://api.coingecko.com/api/v3/coins/markets?vs_currency=USD&order=market_cap_desc&per_page=100&page=1");
     console.log(res.data)
-    setCoins(res.data)
+    const popularCoins = ['btc', 'eth', 'usdt', 'bnb', 'usdc'];
+    const relevantCoins = res.data.filter(coin => {
+        if (popularCoins.findIndex(popularCoin => coin.symbol === popularCoin) < 0){
+        return false
+      } else{
+        return true;
+      }
```
Comment (PkBadger) [inline src/App.js:21]: You can just do

```suggestion
      return popularCoins.findIndex(popularCoin => coin.symbol === popularCoin) >= 0
```

File: src/App.js [hunk 3]
```diff
@@ -24,7 +32,9 @@ function App() {
     <div className="container">
       <h1 className='text-center'>Coin Market</h1>
       <div className='row'>
-        <input type="text" placeholder='Search a Coin' className='form-control bg-dark text-light border-0 mt-4 text-center' onChange={e => setSearch(e.target.value)}/>
+        <input type="text" placeholder='Search a Coin' 
+        className='form-control bg-dark text-light border-0 mt-4 text-center' 
+        onChange={e => setSearch(e.target.value)}/>
```
Comment (PkBadger) [inline src/App.js]: why did this changed?

File: src/components/CoinRow.js [hunk 4]
```diff
@@ -10,14 +10,16 @@ const CoinRow = ({coin, index}) => {
               <spam>{coin.name}</spam>
               <spam className='ms-3 text-muted text-uppercase'>{coin.symbol}</spam>
               </td>
-          <td>{coin.current_price}</td>
+          <td>$ {coin.current_price}</td>
```
Comment (PkBadger) [inline src/components/CoinRow.js:13]: What is this `$` symbol?

File: src/components/CoinRow.js [hunk 5]
```diff
@@ -10,14 +10,16 @@ const CoinRow = ({coin, index}) => {
...
           <td className={
               coin.price_change_percentage_24h > 0 ? "text-success" : "text-danger"
               }
             >
-              {coin.price_change_percentage_24h}
+              $ {coin.price_change_percentage_24h}
           </td>
-          <td>{coin.total_volume}</td>
+          {/* <td>
```
Comment (PkBadger) [inline src/components/CoinRow.js]: Please remove this

File: src/components/TableCoins.js [hunk 6]
```diff
@@ -15,8 +15,8 @@ const TableCoins = ({coins, search}) => {
             </tr>
         </thead>
         <tbody>
-            {filteredCoins.map ((coin, index) => (
-                <CoinRow coin={coin} key={index} index={index +1} />
+            {filteredCoins.map ((coin, index, symbol) => (
+                <CoinRow coin={coin} key={index} index={index +1} symbol={symbol}/>
```
Comment (PkBadger) [inline src/components/TableCoins.js:19]: why is the symbol needed?

File: src/App.js (hunk 1 above)
Comment (Jocelyne28) [reply src/App.js]: done

File: src/App.js (hunk 2 above)
Comment (Jocelyne28) [reply src/App.js:21]: it's necesary?

File: src/App.js (hunk 3 above)
Comment (Jocelyne28) [reply src/App.js]: for fun, i undo it

File: src/components/CoinRow.js (hunk 4 above)
Comment (Jocelyne28) [reply src/components/CoinRow.js:13]: by aesthetics

File: src/components/CoinRow.js (hunk 5 above)
Comment (Jocelyne28) [reply src/components/CoinRow.js]: done

File: src/components/TableCoins.js (hunk 6 above)
Comment (Jocelyne28) [reply src/components/TableCoins.js:19]: by aesthetics

**Comment by Jocelyne28** [comment]
Se hicieron los cambios solicitados

**Pull request merged**